src/foam/nanos/logger/ILogger.java

```diff
@@ -0,0 +1,17 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package foam.nanos.logger;
+
+public interface ILogger {
+  public void start();
```

Comment:
It isn't required to be a NanoService now. Also, you don't declare that it extends NanoService, so having a start() method doesn't mean that it's a NanoService if you don't declare it also.